src/Features/Core/Portable/InitializeParameter/AbstractAddParameterCheckCodeRefactoringProvider.cs

```diff
@@ -144,6 +139,53 @@ internal abstract partial class AbstractAddParameterCheckCodeRefactoringProvider
 
         protected abstract bool CanOffer(SyntaxNode body);
 
+        private async Task<Document> UpdateDocumetForRefactoring(
```

Comment:
`Documet`

Reply:
why was this resolved?

Also, should have `Async` suffix.  Thanks :)

Reply:
Got it! I updated it on my version just haven't merged yet. my bad.